subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/initialization/InstantExecutionStartParameter.kt

```diff
@@ -27,28 +28,26 @@ import java.io.File
 
 class InstantExecutionStartParameter(
     private val buildLayout: BuildLayout,
-    private val startParameter: StartParameter
+    startParameter: StartParameter
 ) {
 
-    val isEnabled: Boolean by unsafeLazy {
-        systemPropertyFlag(SystemProperties.isEnabled)
-    }
+    private
```

Comment:
It seems awkward to have 2 separate ways to communicate the values of the build options internally. Something we should clean up in a follow up PR.